core/src/main/resources/application.properties

```diff
@@ -74,6 +74,9 @@ yona.email.smtp.enableAuth=false
 yona.email.smtp.enableStartTls=false
 yona.email.smtp.username=yona@localhost
 yona.email.smtp.password=
+yona.email.includedMediaBaseUrl=https://app.prd.yona.nu/media/
+yona.email.appleAppStoreLinkUrl=https://itunes.apple.com/us/app/keynote/id361285480?mt=8
+yona.email.googlePlayStoreLinkUrl=https://play.google.com/store/apps/details?id=nu.yona&hl=nl
```

Comment:
This URL includes the locale (NL). Given that the e-mail can be in English and Dutch, I propose to omit the locale. Then the request is handled in browser language.